partials/footer.php

```diff
@@ -1,20 +1,10 @@
-
-<?php
-  $logo = IGV_get_option('_igv_site_options', '_igv_site_logo');
-?>
-  <footer id="footer" class="container">
-    <div class="grid-row">
-<?php
-  if (!empty($logo)) {
-?>
-      <div class="grid-item item-s-6">
-        <img src="<?php echo esc_url($logo); ?>" id="site-logo">
-      </div>
-<?php
-  }
-?>
-      <div class="grid-item item-s-6 font-family-heading font-lighter font-size-larger">
-        COPYRIGHT <?php echo date('Y'); ?>
-      </div>
+<footer id="footer" class="container margin-top-basic margin-bottom-basic">
+  <div class="grid-row justify-between align-items-center">
+    <div class="grid-item">
+      <?php get_template_part('partials/durango-logo'); ?>
```

Comment:
i think it sort of adds another one. just smells wrong to have a file that is just another file with a different encoding. But like it said happy to use this in this project. 

What I remember of the issue was not home/site problems as the reference is theme_url surely? As I remember the issue was on cache based 404 one implementation caused PHP errors, but I remember fixing that but don't think the fix ever got put into the starter pack.